### PR TITLE
Add in-use description to Invoking Operations Section

### PR DESCRIPTION
**Pull request by jdinan**
@nspark Please review for consistency with terms (e.g. completion) you are also working on.

### CODE REVIEW

**Comment by jdinan** [comment]
Closes https://github.com/openshmem-org/specification/issues/241

File: content/execution_model.tex [hunk 1]
```diff
@@ -71,6 +71,16 @@ \subsection{Invoking OpenSHMEM Operations}
...
 
+Buffers provided to \openshmem routines are \emph{in-use} until the
+corresponding \openshmem operation has completed at the calling \ac{PE}.
+Updates to a buffer that is in-use, including updates performed through locally
+and remotely issued \openshmemem operations, result in undefined behavior.
+Similarly, reads from a buffer that is in-use are allowed only when the buffer
+was provided as a \CTYPE{const} argument to the \openshmem routine that it is
+in-use by.  Otherwise, the behavior is undefined. Exceptions are made for
+buffers that are in-use by \acp{AMOs}, as described in
+Section~\ref{subsec:amo_guarantees}}.
```
Comment (jdinan) [inline content/execution_model.tex]: Would it be good to add another sentence like this:
> Please refer to Section~\ref{subsec:quiet} for additional details on the completion of nonblocking operations.

**Comment by RaymondMichael** [comment]
This looks OK to me. You said you wanted a review from Nick. I'll wait for that before merging.

**Comment by jdinan** [comment]
Thanks @RaymondMichael, definitely want to make sure this meshes well with the text that @nspark is working on. Also interested in feedback on the additional sentence I suggested above. Is it helpful to refer readers to sections that specify completion of OpenSHMEM operations?

File: content/execution_model.tex [hunk 2]
```diff
@@ -71,6 +71,16 @@ \subsection{Invoking OpenSHMEM Operations}
 the symmetric address of object $A$ to an \openshmem operation targeting
 \ac{PE} $i$ results in undefined behavior.
 
+Buffers provided to \openshmem routines are \emph{in-use} until the
+corresponding \openshmem operation has completed at the calling \ac{PE}.
+Updates to a buffer that is in-use, including updates performed through locally
+and remotely issued \openshmemem operations, result in undefined behavior.
```
Comment (nspark) [inline content/execution_model.tex]:
```suggestion
and remotely issued \openshmem operations, result in undefined behavior.
```

**Comment by RaymondMichael** [comment]
Yes, I think the new sentence is an important reference.

File: content/execution_model.tex [hunk 3]
```diff
@@ -71,6 +71,16 @@ \subsection{Invoking OpenSHMEM Operations}
...
+Similarly, reads from a buffer that is in-use are allowed only when the buffer
+was provided as a \CTYPE{const} argument to the \openshmem routine that it is
```
Comment (nspark) [inline content/execution_model.tex]:
```suggestion
was provided as a \CTYPE{const}-qualified argument to the \openshmem routine that it is
```

File: content/execution_model.tex [hunk 4]
```diff
@@ -71,6 +71,16 @@ \subsection{Invoking OpenSHMEM Operations}
...
+in-use by.  Otherwise, the behavior is undefined. Exceptions are made for
```
Comment (nspark) [inline content/execution_model.tex]: Could we change "...that it is in-use by" to "for which it is in-use"?

Comment (jdinan) [reply content/execution_model.tex]: Yes! Ending this sentence with a preposition was really bugging me. 🤓

**Comment by jdinan** [comment]
Thank you for the feedback -- please review the updates in e175e34.

**Review by nspark — APPROVED** [review]
I like this text, and I think it better (and more simply) captures the semantic we'll want to use in defining related terms.

**Pull request merged**